packages/app/rollup.config.js

```diff
@@ -1,10 +1,10 @@
 import alias from '@rollup/plugin-alias';
+import { defineConfig } from 'rollup';
 import dts from 'rollup-plugin-dts';
 
 import { externals } from './vite.config.js';
 
-/** @type {import('rollup').MergedRollupOptions} */
-const config = {
+export default defineConfig({
```

Comment:
Saw Cypress recommend this pattern to improve Intellisense and realised that `rollup` supports it as well!